pkg/core/state/validator.go

```diff
@@ -6,13 +6,20 @@ import (
 	"github.com/CityOfZion/neo-go/pkg/util"
 )
 
+// MaxValidatorsVoted limits the number of validators that one can vote for.
+const MaxValidatorsVoted = 1024
+
 // Validator holds the state of a validator.
 type Validator struct {
 	PublicKey  *keys.PublicKey
 	Registered bool
 	Votes      util.Fixed8
 }
 
+// ValidatorsCount represents votes with particular number of consensus nodes
+// for this number to be changeable by the voting system.
+type ValidatorsCount [MaxValidatorsVoted]util.Fixed8
```

Comment:
I don't like this name as it is not immediately obvious what it is (I expect `Count` to be an integer).
Maybe smth like `ValidatorsCountVotes` or similar can be more readable. Thoughts?

Reply:
That's the way it's named in C# implementation and even though it wasn't immediately obvious to me what does it really mean after staring at it for a few hours you're getting used to it. I'm open for suggestions, but I'm not sure `ValidatorsCountVotes` makes it any better.